chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -27,6 +27,13 @@
     </signature-handling>
     <storage/>
   </general>
+  <bootloader>
```

Comment:
I mean to say that this is optional for SLE12, its not explicitly required to have this section.